frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -20,6 +20,7 @@ local VerticalGroup = require("ui/widget/verticalgroup")
 local VerticalSpan = require("ui/widget/verticalspan")
 local WidgetContainer = require("ui/widget/container/widgetcontainer")
 local util = require("util")
+local logger = require("logger")
```

Comment:
Overlooked this but that doesn't look very alphabetical. ^_^
```suggestion
local logger = require("logger")
local util = require("util")
```